src/xcode/ENA/ENA/Source/Models/Submisstion/AntigenTestInformation.swift

```diff
@@ -14,14 +14,19 @@ struct AntigenTestInformation: Codable, Equatable {
 		firstName: String?,
 		lastName: String?,
 		dateOfBirth: Date?
-	) {
+		) {
 		self.hash = hash
 		self.timestamp = timestamp
 		self.firstName = firstName
 		self.lastName = lastName
 		self.dateOfBirth = dateOfBirth
+		guard let dateOfBirth = dateOfBirth else {
+			dateOfBirthString = nil
+			return
```

Comment:
Log something when return in guard

Reply:
but this is not an error, we just set the property accordingly

Reply:
ah did not see because of github's reduction for review windows.